src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Editor/CompositorWindowBase.cs

```diff
@@ -169,7 +169,7 @@ protected void HolographicCameraNetworkConnectionGUI(string deviceTypeLabel, Dev
 
                 if (GUILayout.Button(new GUIContent("Locate Shared Spatial Coordinate", "Detects the shared location used to position objects in the same physical location on multiple devices")))
                 {
-                    SpatialCoordinateSystemManager.Instance.InitiateRemoteLocalization(deviceInfo.ConnectedEndpoint, MarkerDetectorSpatialLocalizer.Id, new MarkerDetectorLocalizationSettings
+                    CompositorWorldAnchorLocalizationManager.Instance.RunRemoteLocalizationWithWorldAnchorPersistence(spatialCoordinateSystemParticipant, MarkerDetectorSpatialLocalizer.Id, new MarkerDetectorLocalizationSettings
                     {

```

Comment:
Do most folks always want this behavior or do we want the ability to dynamically turn on/off the persisting/restoring of anchors?

Reply:
Currently our internal system always persists and restores anchors. I believe this was also the default behavior of the original SpectatorView, and I haven't heard requests not to have this behavior.

---
In reply to: [295550861](https://github.com/microsoft/MixedReality-SpectatorView/pull/26#discussion_r295550861) [](ancestors = 295550861)